analyzers/src/SonarAnalyzer.Common/Common/RegexConstants.cs

```diff
@@ -22,5 +22,5 @@ namespace SonarAnalyzer.Common;
 
 public static class RegexConstants
 {
-    public static TimeSpan DefaultTimeout => TimeSpan.FromMilliseconds(100);
+    public static TimeSpan DefaultTimeout => TimeSpan.FromMilliseconds(200);
```

Comment:
Actually, with the morning and a fresh mind, I see the issue differently. Shouldn't we increase the timeout for only the complicated regex that had timeouts on the ITs and not all of them?